scripts/linz-bde-schema-load.bash

```diff
@@ -147,26 +147,26 @@ if test "${ADD_REVISIONS}" = "yes" -a $TABLEVERSION_SUPPORTS_STDOUT = yes; then
     ${TABLEVERSION_LOADER} ${TABLEVERSION_OPTS} - || rollback
 fi
 
-if test $PGDATABASE != "-"; then
+if test "$PGDATABASE" != "-"; then
     echo 'SET client_min_messages TO WARNING;'
 fi
 
 cat << EOF
 CREATE EXTENSION IF NOT EXISTS postgis SCHEMA public;
 EOF
 
-for file in ${SCRIPTSDIR}/*.sql; do
+for file in "${SCRIPTSDIR}"/*.sql; do
     if test ${SKIP_INDEXES} = 'yes' &&
-       `basename $file .sql` = '04-bde_schema_index';
+       [ "$(basename "$file" .sql)" = '04-bde_schema_index' ];
```

Comment:
Same as [elsewhere](https://github.com/linz/linz-bde-schema/pull/232#discussion_r796113147), this is probably a bug.